scripts/helper_find_llvm_install.sh

```diff
@@ -0,0 +1,25 @@
+# Constant: location of the custom tool binaries in the dev container
```

Comment:
I was confused by the missing shbang in a .sh file.  Maybe change the extension to .inc to indicate that this should not be run directly?  Or move it to an `includes` directory?

Reply:
Good point, I will move it to an `includes` directory. I tried `.inc`, but it prevents vim from correctly detecting the file as a shell script for syntax highlighting.